doc/manuals/orion-api.md

```diff
@@ -2157,26 +2161,14 @@ Some notes to take into account when using `json` instead of `payload`:
 * `payload` and `json` cannot be used at the same time
 * `Content-Type` header is set to `application/json`, except if overwritten by `headers` field
 
-Some considerations to take into account when using custom notifications:
+### Omitting payload
 
-* It is the client's responsibility to ensure that after substitution, the notification is a
-  correct HTTP message (e.g. if the Content-Type header is application/xml, then the payload must
-  correspond to a well-formed XML document). Specifically, if the resulting URL after applying the
-  template is malformed, then no notification is sent.
-* In case the data to notify contains more than one entity, a separate notification (HTTP message)
```

Comment:
No longer a possiblity. Since the removal of initial notification, all the notification are mono-entity.

NTC (just to explain why this paragraph has been deleted)